### DISCUSSION TITLE
Primevue Tree component

### DISCUSSION THREAD

**Discussion by AkshayMahM** [primefaces/community#1998] [PrimeVue] [answered]
How to use tree component custom passthrough(pt) option method.  
![Screenshot 2024-05-21 185527](https://github.com/primefaces/community/assets/88873410/891fbb03-e647-4871-b69a-1e547af50cf3)

**Comment by tugcekucukoglu** [comment, marked as answer]
https://tailwind.primevue.org/tree/#presets.lara

**Comment by AkshayMahM** [comment]
Thank you 👍
